src/drawing/strokeactions.ts

```diff
@@ -17,8 +17,10 @@ let tid: number | NodeJS.Timeout = 0
  * Start the current stroke when mouse is pressed down
  * @param {*} point
  */
-export function startLiveStroke(point: Point): void {
-    store.dispatch(START_LIVESTROKE([point.x, point.y]))
+export function startLiveStroke(point: Point, pageId: string): void {
+    const liveStroke = getLiveStroke()
+    liveStroke.start(point, pageId)
```

Comment:
Is this safe to call a method that mutates an object in the store? (Outside of a reducer)